lib/fluent/test/startup_shutdown.rb

```diff
@@ -0,0 +1,31 @@
+#
```

Comment:
My private memo: this code should be included in v0.12 test driver and v0.14 test driver (w/ plugin helper).